eng/doc/fips/README.md

```diff
@@ -360,9 +360,11 @@ This feature does not require any additional configuration, but it only works wi
 
 ### TLS with FIPS-approved settings
 
-The Go TLS stack will automatically use OpenSSL crypto primitives when running in FIPS mode. Yet, the FIPS 140-2 standard places additional restrictions on TLS communications, mainly on which cyphers and signers are allowed.
+The Go TLS stack will automatically use OpenSSL crypto primitives. Yet, the FIPS 140-2 standard places additional restrictions on TLS communications, mainly on which cyphers and signers are allowed.
 
-A program can import the `crypto/tls/fipsonly` package to configure the Go TLS stack so it is compliant with these restrictions. The configuration is done by an `init()` function, so only importing it is necessary:
+Since Go 1.22, the Microsoft Go runtime automatically enforces that `crypto/tls` and `crypto/x509` only use FIPS-approved settings when running in FIPS mode. This differs from upstream's BoringCrypto backend, which requires you to import `crypto/tls/fipsonly` to apply the FIPS-mandated restrictions. The Microsoft Go crypto backends do this automatically to reduce the source code changes necessary to produce a FIPS-compliant Go application.
```

Comment:
I think it's worth adding something in the intro to clarify that even though boringcrypto provides `import _ "crypto/tls/fipsonly"` and you'd need to add that to your project, you don't need to with Microsoft Go.

(We could potentially remove that intro paragraph entirely, although I think we need to be careful that people using Go <1.22 still easily see/understand that they need the import.)

Reply:
I've updated the `Usage: Common configurations` section to mention this change too.